cli/src/commands/codegen.ts

```diff
@@ -45,6 +70,16 @@ export const codegen = (flags: CodegenFlags) => {
   } else {
     deleteOldGeneratedFiles(outDir);
   }
+
+  if (flags.appId && flags.language) {
+    return generateTypings(
+      outDir,
+      flags.language,
+      flags.appId,
+      flags.staging ?? false,
+    );
+  }
+
   const absoluteAsProtoGenPath = path.join(baseDir, AS_PROTO_GEN_PATH);
```

Comment:
I think we should export the code below the `return` to a function like `generateAssemblyScriptPluginTypings` and rename `generateTypings` to `generateClientTypings` to make the functionality clear when having a first look at the code

Reply:
Addressed